raiden/transfer/mediated_transfer/events.py

```diff
@@ -186,7 +186,7 @@ def __str__(self):
 
     def __eq__(self, other):
         return (
-            isinstance(other, SendRevealSecret) and
+            isinstance(other, SendRevealSecret2) and
```

Comment:
Is this an oversight or do you want to be compatible with the old state classes? (Asking because the same happens in the mediator PR)

Reply:
oversight, definitely